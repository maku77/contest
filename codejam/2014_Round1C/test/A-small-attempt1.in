100
3/32
167/982
57/128
3/4
9/32
165/512
145/593
12/215
1/16
15/128
1/128
7/8
97/892
357/418
435/917
3/128
1/2
503/821
5/8
29/32
3/256
61/512
1/16
3/16
55/128
5/16
1/2
1/128
5/8
47/254
1/32
7/8
9/128
1/128
1/256
13/128
114/161
1/8
13/16
1/16
1/64
1/256
1/16
55/61
339/448
1/8
211/512
1/2
1/2
59/64
1/4
1/32
1/128
1/8
1/16
1/16
11/64
139/328
1/32
171/236
1/32
51/128
15/16
1/128
1/4
1/2
96/521
1/128
1/32
233/394
3/4
7/512
3/256
706/987
3/16
1/512
467/862
1/2
13/32
1/2
1/4
5/16
119/512
1/16
206/231
1/128
1/16
1/8
1/128
1/64
107/256
1/16
1/128
1/2
71/83
1/64
467/856
1/256
1/16
1/128
