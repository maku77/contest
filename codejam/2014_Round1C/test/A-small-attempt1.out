Case #1: 4
Case #2: impossible
Case #3: 2
Case #4: 1
Case #5: 2
Case #6: 2
Case #7: impossible
Case #8: impossible
Case #9: 4
Case #10: 4
Case #11: 7
Case #12: 1
Case #13: impossible
Case #14: impossible
Case #15: impossible
Case #16: 6
Case #17: 1
Case #18: impossible
Case #19: 1
Case #20: 1
Case #21: 7
Case #22: 4
Case #23: 4
Case #24: 3
Case #25: 2
Case #26: 2
Case #27: 1
Case #28: 7
Case #29: 1
Case #30: impossible
Case #31: 5
Case #32: 1
Case #33: 4
Case #34: 7
Case #35: 8
Case #36: 4
Case #37: impossible
Case #38: 3
Case #39: 1
Case #40: 4
Case #41: 6
Case #42: 8
Case #43: 4
Case #44: impossible
Case #45: impossible
Case #46: 3
Case #47: 2
Case #48: 1
Case #49: 1
Case #50: 1
Case #51: 2
Case #52: 5
Case #53: 7
Case #54: 3
Case #55: 4
Case #56: 4
Case #57: 3
Case #58: impossible
Case #59: 5
Case #60: impossible
Case #61: 5
Case #62: 2
Case #63: 1
Case #64: 7
Case #65: 2
Case #66: 1
Case #67: impossible
Case #68: 7
Case #69: 5
Case #70: impossible
Case #71: 1
Case #72: 7
Case #73: 7
Case #74: impossible
Case #75: 3
Case #76: 9
Case #77: impossible
Case #78: 1
Case #79: 2
Case #80: 1
Case #81: 2
Case #82: 2
Case #83: 3
Case #84: 4
Case #85: impossible
Case #86: 7
Case #87: 4
Case #88: 3
Case #89: 7
Case #90: 6
Case #91: 2
Case #92: 4
Case #93: 7
Case #94: 1
Case #95: impossible
Case #96: 6
Case #97: impossible
Case #98: 8
Case #99: 4
Case #100: 7
